src/gd_avif.c

```diff
@@ -540,14 +546,25 @@ static avifBool _gdImageAvifCtx(gdImagePtr im, gdIOCtx *outfile, int quality, in
 	// Note that MATRIX_COEFFICIENTS_IDENTITY enables lossless conversion from RGB to YUV.
 
 	avifImage *avifIm = avifImageCreate(gdImageSX(im), gdImageSY(im), 8, subsampling);
-
+#if AVIF_VERSION >= 1000000
```

Comment:
To clarify my comment above: we could drop this version check, since `avifImageCreate()` always returned a pointer. Same for `avifEncoderCreate()` below.

Reply:
anywhere we can drop CPP `#if` logic is an improvement

Reply:
Done.